keras_hub/src/models/cspnet/cspnet_backbone.py

```diff
@@ -357,18 +357,6 @@ def apply(x):
             dtype=dtype,
             name=f"{name}_bottleneck_block_bn_3",
         )(x)
-        if activation == "leaky_relu":
```

Comment:
Did we not need these ever? Fine to remove, just not sure what changed.

Reply:
This was not required and was not called till I started porting new weights, previous one used other blocks, as per the timm architecture for the last BN, activation is not applied in bottleneck block.

Reply:
Sg!